crate/server/src/database/redis/redis_with_findex.rs

```diff
@@ -228,6 +228,10 @@ impl Database for RedisWithFindex {
         None
     }
 
+    async fn migrate(&self, _params: Option<&ExtraDatabaseParams>) -> KResult<()> {
+        Ok(())
+    }
```

Comment:
maybe add a TODO, marking that it is not implemented for Redis - or better, raise an Error ?

Reply:
replaced by:
```
    async fn migrate(&self, _params: Option<&ExtraDatabaseParams>) -> KResult<()> {
        unimplemented!("Redis-with-Findex does not support migrate operation");
    }
```